### ISSUE TITLE
Questions about the test accuracy of only 0.8

### ISSUE THREAD

**Issue by hemo0826** [Dootmaan/DTFD-MIL.PyTorch#4]
Hello, you have done an excellent job. We have encountered some problems in surfacing your work. There are frequent interruptions of downsample error and many error patches when extracting slices, so there may be errors when extracting slice features accordingly. So we would like to know if you can share the npy file of camelyon16 after feature extraction? We are looking forward to your reply!

**Comment by Dootmaan** [comment]
Hi @hemo0826 and thank you for your question. You test set performance seems to be very different from the results in issue #2. But anyway, I will upload the preprocessed .npy files later.

**Comment by Dootmaan** [comment]
The preprocess dataset is about 16GB and currently I have run out of Google Drive space. I will find an alternative platform to upload these files, and when the uploading is done you will receive a notification.

**Comment by titizheng** [comment]
Hello, first of all, thank you for your insights. However, I'm currently facing a situation where I'm achieving a maximum accuracy of only 0.81 on the 129 test samples in the 16-challenge. I've used the data preprocessing methods provided by CLAM, and I'm using a pre-trained ResNet50 on ImageNet to extract features. Would it be possible for you to share the trained model parameters after your training? I'd like to further test to see if the issue lies with the data. Looking forward to your response, and thank you very much.
